tests/test_study_definition.py

```diff
@@ -54,3 +54,13 @@ def test_unrecognised_database_url_raises_error(tmp_path, monkeypatch):
             sex=patients.sex(),
             age=patients.age_as_of("2020-01-01",),
         )
+
+
+def test_errors_are_triggered_without_database_url(tmp_path, monkeypatch):
+    monkeypatch.delenv("DATABASE_URL", raising=False)
+    with pytest.raises(Exception):
```

Comment:
Can we check that the expected exception is raised?

Reply:
Ah yes, fair point.